### DISCUSSION TITLE
Eleccion de proyecto

### DISCUSSION THREAD

**Discussion by JoseUrrego123** [rramosp/ai4eng.v1#280] [00 - General]
Hola profe, buenos dias.

Mis compañeros y yo estamos en la búsqueda de un datasets que nos permita desarrollar con éxito el curso y queremos enfocarlo a la parte comercial, hemos encontrado dos, uno trata de la predicción de ventas de una serie de tiendas en ecuador y otro consiste en una serie de farmacias que operan en Europa, no sabemos con certeza si cumplen con los requerimientos del curso, nos podrias porfa indicar si alguno de estos dos datasets (o los dos) cumplen con los parametros?

https://www.kaggle.com/competitions/rossmann-store-sales
https://www.kaggle.com/competitions/store-sales-time-series-forecasting/overview

Muchas gracias

**Comment by Felipe-RA** [comment]
Saludos @JoseUrrego123 

Rossman Sales no cumplé el requisito de dimensionalidas (30 columnas minimo y tiene 9)

Store Sales - Time series forecasting tampoco lo cumple (sólo tiene 6)

Los invito a seguir buscando mientras tienen en cuenta este y los otros tres requisitos listados en la página del curso.

Hasta pronto!

**Comment by JoseUrrego123** [comment]
Listo profe, pensé que de pronto me servía, como vi que decía 29 columnas.

**Reply by JoseUrrego123** [reply]
Profe, tengo una pregunta, la cantidad de columnas es con solo un archico csv que conforme el datasets o las columnas en su totalidad de dicho datasets?

**Reply by Felipe-RA** [reply]
Ten cuidado, el summary que hace Kaggle es una suma de todas las columnas en todos los datasets.

Esas 30 columnas deben ser de tu archivo de entrenamiento (usualmente listado como train , data, dataset, u otros nombres) 

Archivos como sample_submissions, data_dict no hacen parte de las columnas que puedes usar para entrenar, sino que son ficheros informativos.

Si llegas a ver un archivo test, es para que pruebes tus modelos sobre esos datos.

Hay casos en los que tu dataset de entrenamiento si que puede estar partido en múltiples archivos, y se espere que los unifiques para el posterior entrenamiento. Esto se sabe cuando ganas entendimiento de lo que pide la competencia.

Echale un vistazo al contenido del curso relacionado a "como competie en Kaggle" si quieres saber a fondo cómo va esto 

Hasta pronto
